### DISCUSSION TITLE
I need help with blowfish

### DISCUSSION THREAD

**Discussion by mcincura** [phantom/docs#249] [Help & Support] [answered]
### Summary

I am developing a new SPL token and I want to have a presale functionality on our website. The blowfish is blocking our domain and I need someone to vouch for me and my project. I can provide the source code for everything so you can verify no malicious intent. All of the code is currently set to devnet for testing purposes. Feel free to check out the issue by yourself on my website. 
As of right now the website is not correctly working with the mobile phantom app.

### Example

trevor-solana.com

### Steps to Reproduce

![phantom wallet error](https://github.com/phantom/docs/assets/171041308/b7bda8bf-45fe-4eab-9ce2-8d27944d7258)

### Is there an existing discussion for this?

- [X] I have searched the existing discussions

**Comment by adamdelphantom** [comment, marked as answer]
Hello @mcincura,

Following up with Blowfish and making whatever resources they ask for is your best path for having the warning removed. If you're still on devnet and testing you should be able to test your transactions by bypassing the warning for the time being.

**Comment by ghost** [comment]
I am having a similar issue... I'll lyk if I find a solution. We have on boarded a new Dev company that are also looking into this, we are going for a resolution soon.
